src/components/Header.vue

```diff
@@ -8,20 +8,11 @@
                 class="image__image"
             />
         </div>
-
     </section>
-
 </template>
 
-<script lang='ts'>
-import { defineComponent } from 'vue';
-
-export default defineComponent({
-
-});
-</script>
-
 <style scoped lang='scss'>
+// duplicación
```

Comment:
Repites en algunas partes css, si sigues alguna arquitectura como por ejemplo  [atomic design](https://vuedose.tips/how-to-structure-a-vue-js-app-using-atomic-design-and-tailwindcss) te puede ayudar a evitar esto y mejorar el mantenimiento del código.

Reply:
Además luego es más sencillo documentar tus componentes con https://storybook.js.org/